internal/kafka/internal/services/clusters_test.go

```diff
@@ -2340,3 +2475,406 @@ func Test_ClusterService_GetExternalID(t *testing.T) {
 		})
 	}
 }
+
+func Test_clusterService_GetClientId(t *testing.T) {
+	type fields struct {
+		connectionFactory      *db.ConnectionFactory
+		clusterProviderFactory clusters.ProviderFactory
+	}
+	type args struct {
+		clusterId string
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		args    args
+		want    string
+		wantErr bool
+		setupFn func()
+	}{
+		{
+			name: "Should return the clusters client ID",
+			fields: fields{
+				connectionFactory:      db.NewMockConnectionFactory(nil),
+				clusterProviderFactory: &clusters.ProviderFactoryMock{},
+			},
+			args: args{
+				clusterId: testClusterID,
+			},
+			setupFn: func() {
+				res := []map[string]interface{}{
+					{
+						"id":          "testid",
+						"cluster_id":  testClusterID,
+						"cluster_dns": "",
+						"client_id":   "ClientID",
+					},
+				}
+				mocket.Catcher.Reset().
+					NewMock().WithQuery(`SELECT * FROM "clusters" WHERE "clusters"."cluster_id" = $1`).
+					WithArgs(testClusterID).
+					WithReply(res)
+			},
+			wantErr: false,
+			want:    "ClientID",
+		},
+		{
+			name: "Should return empty string if cluster is nil",
```

Comment:
This description is too implementation specific.
A better description would be:
`An empty string is returned if the provided cluster is not found`